war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -80,13 +80,12 @@ public void setOrganizationCode(final String organizationCode) {
    }
 
    /**
-    * Reads a map of all organizations in the system.
+    * Reads  all organizations in the system.
     *
-    * @return map of values (code, name) of all organizations in the system
+    * @return list of organizations
     */
-   public Map<String, String> readOrganizationsMap() {
-      List<DataDocument> documents = dataStorage.search(LumeerConst.Organization.COLLECTION_NAME, null, Arrays.asList(LumeerConst.Organization.ATTR_ORG_NAME, LumeerConst.Organization.ATTR_ORG_CODE));
-      return documents.stream().collect(Collectors.toMap(d -> d.getString(LumeerConst.Organization.ATTR_ORG_CODE), d -> d.getString(LumeerConst.Organization.ATTR_ORG_NAME)));
+   public List<DataDocument> readOrganizations() {
+      return dataStorage.search(LumeerConst.Organization.COLLECTION_NAME, null, null, 0, 0);
```

Comment:
Should not we return DTO here as well instead of providing the database entry structure to the public?